server/db/models/product.js

```diff
@@ -27,6 +27,10 @@ const Product = db.define("product", {
     defaultValue:
       "https://static.vecteezy.com/system/resources/previews/001/196/554/large_2x/person-png.png",
   },
+  inStock: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: true,
```

Comment:
inStock is an integer, not a boolean.
```suggestion
    type: Sequelize.INTEGER,
    validate: {
        min: 0,
    },
```